scripts/custom-screens/CustomDataDisplayComponents.tsx

```diff
@@ -1,52 +1,60 @@
 import {registerScreen} from "@haulmont/jmix-react-web";
-import styles from "../App.module.css";
-import {Card} from "antd";
+import {Card, Space} from "antd";
 import { EntityHierarchyTree } from "@haulmont/jmix-react-antd";
+import { Label } from "@haulmont/jmix-react-web"
```

Comment:
better to import from existing `@haulmont/jmix-react-web` which is placed upper.